homepage/homepage/content/docs/using-covalues/colists.mdx

```diff
@@ -182,17 +182,17 @@ export function getCurrentTasks(list: co.loaded<ListOfTasks>) {
 
 async function main() {
   const myTaskList = ListOfTasks.create([]);
```

Comment:
Is there a specific reason we're creating the Task CoValues manually here rather than just pushing JSON to the list? Might make things a bit more concise otherwise.